src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -370,6 +367,8 @@ ucs_status_t uct_rc_mlx5_ep_am_zcopy(uct_ep_h tl_ep, uint8_t id, const void *hea
         UCT_TL_EP_STAT_OP(&ep->super.super, AM, ZCOPY,
                           header_length + uct_iov_total_length(iov, iovcnt));
         UCT_RC_UPDATE_FC(&iface->super, &ep->super, id);
+        ucs_assert(uct_rc_iface_check_pending(&iface->super,
+                                              &ep->super.arb_group));
```

Comment:
maybe move UCT_RC_CHECK_RES outside of uct_rc_mlx5_ep_zcopy_post, and in line 359 replace by UCT_RC_CHECK_RES_AND_FC ?